service/registry/src/main/java/io/helidon/service/registry/Interception.java

```diff
@@ -130,4 +130,12 @@ interface Chain<V> {
             V proceed(Object[] args) throws Exception;
         }
     }
+
+    /**
+     * Interceptor for a specific element and annotation.
+     * Implementations of this interface are {@link io.helidon.service.registry.Service.Named} by the
+     * fully qualified target type + "." + element signature.
+     */
+    public interface ElementInterceptor extends Interceptor {
```

Comment:
Just making sure you want to introduce a new type in the service registry as part of your fault tolerance PR.